qiskit/providers/ibmq/api_v2/rest/schemas/job.py

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2017, 2019.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Schemas for job."""
+from marshmallow.validate import OneOf
+from qiskit.providers.ibmq.apiconstants import ApiJobStatus
+from qiskit.validation import BaseSchema
+from qiskit.validation.fields import String, Url, Nested, Number, Boolean
+
+
+# Helper schemas.
+
+class InfoQueueResponseSchema(BaseSchema):
+    """Nested schema for StatusResponseSchema"""
+
+    # Required properties
+    status = String(required=True)
+    position = Number(required=True, missing=0)
+
+
+class JobResponseSchema(BaseSchema):
```

Comment:
If this serves strictly as a nested schema for `CallbackUploadResponseSchema`, then `JobResponseSchema` is not a very good name since there are a lot fo `/Jobs` related endpoints (and hence responses)

Reply:
Hmmm, yes I agree. Maybe we should come up with some better naming policy? Nothing good comes to mind.

Reply:
how about `CallbackUploadResponseJobSchema` for now? we'll likely combine this with other response schemas later since multiple endpoints return the same "job" results.

Reply:
`CallbackUploadResponseNestedJobSchema`?